opensearch/opensearch-kafka/readme.md

```diff
@@ -0,0 +1,55 @@
+# OpenSearch Example
+
+This repo is intended to provide the SA team with a means to populate an OpenSearch index with data so that you can demonstrate how OpenSearch can be used.
+
+## Requirements
+
+- [Terraform CLI](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli)
+- Python 3.8+
+- [Aiven CLI](https://docs.aiven.io/docs/tools/cli.html)
+- Project created via Aiven Console or CLI for running services
+- (Optional) [Postman v2.1](https://www.postman.com/downloads/)
+
+
+## Steps
+1. Run the setup.sh file to unzip the file ``` ./setup.sh ```
+2. Open a new terminal/shell window and run the following command to save the Aiven API token to an ENV variable
+```
+. ../scripts/save-token.sh
+```
+3. Run the provided terraform files to create an OpenSearch instance and a Kafka service we will use to add data from the csv file.
+```
+cd terraform
+terraform init
+terraform plan
+terraform apply -var="aiven_api_token=$AVN_TOKEN" -var="aiven_project=<project-name>"
+```
+4. Run the following shell command to create connection between Opensearch and Kafka. As of 12/8/2022 there is a bug in the Aiven console which prevents setting the value converter field correctly. There may also be a way to make this work via terraform but will have to be added in future versions.
```

Comment:
nice! yes, I've ran into several KC config issues on the console.

Reply:
thank you